<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentwindow</class>
 <widget class="QDialog" name="studentwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Dashboard</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>1001</width>
       <height>631</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>60</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Routine</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>961</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="routine_day">
          <item>
           <property name="text">
            <string>Sunday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Monday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tuesday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Wednesday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Thursday</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Friday</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Faculty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="routine_faculty"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="routine_year">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Semester</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="routine_semester">
          <item>
           <property name="text">
            <string>I</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>II</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="routine_table">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>41</y>
         <width>991</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="routine_show">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>560</y>
         <width>931</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
